src/terms/Hamiltonian.jl

```diff
@@ -0,0 +1,96 @@
+### A Hamiltonian is composed of blocks (kpoints), which have a list of RFO corresponding to each term
+# This is the "high-level" interface, provided for convenience
+using Printf
+
+struct HamiltonianBlock
+    basis::PlaneWaveBasis
+    kpoint::Kpoint
+    # The Hams are vectors of RealFourierOperator, not typed because of type invariance issues.
+    Hams::Vector # the original list of RFOs (as many as there are terms), kept for easier exploration
+    optimized_Hams::Vector # the optimized list of RFOs, to be used for applying
```

Comment:
Two spaces before `#`.
Why not just call them operators and optimized_operators?